cockpit/src/main/java/com/polidea/cockpit/paramsedition/viewholder/ColorParamViewHolder.kt

```diff
@@ -0,0 +1,52 @@
+package com.polidea.cockpit.paramsedition.viewholder
+
+import android.graphics.Color
+import android.graphics.drawable.GradientDrawable
+import android.support.annotation.ColorInt
+import android.support.v4.content.ContextCompat
+import android.support.v7.widget.AppCompatEditText
+import android.view.View
+import com.polidea.cockpit.R
+import com.polidea.cockpit.core.CockpitParam
+import com.polidea.cockpit.core.type.CockpitColor
+import com.polidea.cockpit.utils.TextWatcherAdapter
+import java.lang.RuntimeException
+
+internal class ColorParamViewHolder(view: View) : ParamBaseValueWithRestoreViewHolder<CockpitColor>(view) {
+
+    private val value: AppCompatEditText = view.findViewById(R.id.cockpit_color_param_value)
+
+    private val colorPreview: View = view.findViewById(R.id.cockpit_color_preview_view)
```

Comment:
We should create ColorPreviewView and place there all the logic which changes its drawable